referrer-policy/get_failing_tests.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import json
+
+data = json.load(open("attr-referrer-failing.json"))
+
+prefix = "http://web-platform.test:8000"
```

Comment:
Yes, sry about this one, it slipped through (because of the manifest mechanism, which is quite bad for development), I've used it to debug my Chromium CL and it helped. :)